required/tools/calc.dtx

```diff
@@ -1306,7 +1306,7 @@
 % \changes{v4.0e}{1997/11/11}
 %    {typo fixed}
 % If |\calc@post@scan| reads a character that is not one of `\texttt{+}',
-% `\texttt{-}', `\texttt{*}', `\texttt{/}', or `\texttt{)}', an error
```

Comment:
Actually, _and_ is technically correct because this is like saying “one of these five possibilities”. What are the five possibilities? `\texttt{+}', `\texttt{-}', `\texttt{*}', `\texttt{/}', and (!) `\texttt{)}'.

“_A_ or _B_” is equivalent to “one of _A_ and _B_”. You can use _or_ by omitting the _one of_ part.

Reply:
Yes, you are correct (my logic was never great:-)!

But, therefore, maybe removing the "one of" would be good?

Reply:
I would use  "that is neither ... nor ..." but that is just my taste